### PR TITLE
GE Export: Support quality definition stored in a file

### PR DESCRIPTION
**Pull request by saugerDecathlon**
#136

### CODE REVIEW

**Comment by simonharrer** [comment]
Did you have a look at the resolve part that we already use for the definitions?

File: tests/examples/great-expectations/datacontract_quality_file.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      order_id:
+        type: string
+        required: true
+      processed_timestamp:
+        type: timestamp
+        required: true
+quality:
+  type: great-expectations
+  specification:
+    orders: "$ref: ./examples/great-expectations/quality.json"
```
Comment (simonharrer) [inline tests/examples/great-expectations/datacontract_quality_file.yaml:23]: I'd have expected the following:
```yaml
quality:
  type: great-expectations
  specification:
    orders: 
       $ref: './examples/great-expectations/quality.json'
```

Comment (saugerDecathlon) [reply tests/examples/great-expectations/datacontract_quality_file.yaml]: updated

**Comment by saugerDecathlon** [comment]
> Did you have a look at the resolve part that we already use for the definitions?

Indeed, I did not see it. I have now added a the resolve ref quality function in the resolve.py

**Comment by simonharrer** [comment]
Can you add an include_quality option for the resolve_data_contract_from_str please?

**Comment by simonharrer** [comment]
Can you use $ref instead of ref

**Comment by jochenchrist** [comment]
Thanks for your contribution. I merged https://github.com/datacontract/datacontract-specification/pull/54

**Comment by saugerDecathlon** [comment]
> Thanks for your contribution. I merged [datacontract/datacontract-specification#54](https://github.com/datacontract/datacontract-specification/pull/54)

Thanks, the tests have passed

**Pull request merged**